sympy/physics/vector/tests/test_vector.py

```diff
@@ -169,3 +169,11 @@ def test_vector_simplify():
     test4 = ((-4 * x * y**2 - 2 * y**3 - 2 * x**2 * y) / (x + y)**2) * N.x
     test4 = test4.simplify()
     assert (test4 & N.x) == -2 * y
+
+
+def test_vector_evalf():
+    a, b = symbols('a b')
+    v = 5 * A.x
+    assert v.evalf() == 5.0 * A.x
```

Comment:
A better test might be `v = pi *A.x` and then evaluating to specific precision, where `pi` is a sympy defined constant.